cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -101,21 +101,32 @@ Deploying the Example
 Since deploying the WikipediaPipelineApp involves loading an artifact and creating an application from it, the
 preferred method of deploying it is to use the CDAP CLI.
 
-- Load the Artifact::
+- Load the Artifact:
 
-    $ cdap-cli.sh load artifact target/WikipediaPipeline-3.3.0-SNAPSHOT.jar name WikipediaPipelineApp version
-    3.3.0-SNAPSHOT
-    Successfully added artifact with name 'WikipediaPipelineApp'
+  .. container:: highlight
+
+    .. parsed-literal::
+    
+      |$| cdap-cli.sh load artifact target/WikipediaPipeline-\ |release|.jar name WikipediaPipelineApp version |release|

```

Comment:
I tried it, and it turns out that because of the hyphen, the backslash space isn't needed, and the replacement is rendered correctly. If it was ``WikipediaPipeline|release|.jar``, that wouldn't work...but we don't have that.